src/app.py

```diff
@@ -1076,7 +1076,7 @@ def create_entity(entity_type):
             'previous_revision_uuid', 
             'next_revision_uuid'
         ]
-    elif normalized_entity_type in ['Upload', 'Collection']:
+    elif normalized_entity_type in ['Upload', 'Collection', 'Epicollection']:
```

Comment:
There's another place needs this same update: https://github.com/hubmapconsortium/entity-api/blob/5c62c7b6a8852621a20e4db55f299857b51d8a97/src/app.py#L1359